composer.json

```
@@ -17,13 +17,13 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
         "spatie/laravel-package-tools": "^1.4.3",
         "illuminate/support": "^10.0|^9.0"
     },
     "require-dev": {
         "brianium/paratest": "^6.2",
-        "livewire/livewire": "^2.4",
+        "livewire/livewire": "^3.0@beta",
```

```suggestion
        "livewire/livewire": "^3.0",
```
Livewire is no longer in beta.